docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Code Builder Quick Start
+lang: en
+---
+
+## Overview
+Complete this Quick Start to become familiar with the Code Builder interface and perform some simple tasks in your Code Builder environment.
+## Important Considerations for Code Builder Beta
+We've capped usage for beta at 20 hours for a maximum of 30 days. We highly recommend that you save your work and close the browser tab that is running Code Builder to stop the usage clock when you aren’t using Code Builder.
+
+### Don’t Forget to Save Your Work
+Working in the cloud has its advantages. There are some differences though. Unlike working on a desktop where you save your files to a local machine, you must either deploy your changes to your org, or commit them to source control to save your work. Remember to do this before you close the Code Builder tab on your browser so you don’t lose your work.  Throughout the beta, Code Builder environments could be deleted.  All beta environments will be removed before GA.
+
+
+## Let’s Get Started
+To get started you’ll do the following:
+1. Create a Code Builder Environment.
+2. Create a Salesforce DX project.
+3. Connect to your Org by logging in.
+4. Now you’re ready to use Code Builder.
+
+## Connect to Your Dev Environment
+1. From the App Launcher, find and open **Code Builder**.
+(If you don't see Code Builder as an option, or you see an empty screen when you launch it, contact your admin to make sure you have the correct license and permission set assigned to you.)
```

Comment:
Note instead of parens?

Reply:
But that's a great note to add. Thinking more about access to the Dev Hub, if they aren't admins - then they need the SFDX perm set (which isn't a formal thing but documented in the Dev Guide). If you have no idea what I'm saying, we can talk in realtime.